english/escape-to-the-wilderness/wilderness.md

```diff
@@ -0,0 +1,22 @@
+You decide to escape the city for a weekend and head out to the wilderness.
+
+With your bag all packed with camping gear, you make your way toward the mountains.
+
+The hike to a campsite was relatively easy so you settle down in a sleeping bag 
+next to the campfire, and drift off to sleep. 
+
+You awake to find you're being dragged across the forest floor. But by who or what?
+
+Perhaps a [bear](https://github.com/udacity/create-your-own-adventure/blob/master/english/bear/bear.md) got a hold of you.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.